src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -127,6 +132,10 @@ private async Task ProcessRequestCoreAsync(StreamingMessage request)
             {
                 responseMessage.WorkerInitResponse = WorkerInitRequestHandler(request.WorkerInitRequest);
             }
+            else if (request.ContentCase == MsgType.WorkerTerminate)
```

Comment:
I think this whole block would refactor nicely into a switch statement. And does order matter here? Do we want to put the terminate statement first?

Reply:
Order does not matter. Agree on the _switch_ to switch.

Reply:
Ah it's by type!! My bad :D